.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
```

Comment:
Anyway lint workflows has been added here https://github.com/kodingdotninja/create-chakra-icons/pull/12

but looking for feedback @grikomsn

Reply:
I'll probably remove this and use #12.